autogluon/utils/tabular/ml/models/abstract/abstract_model.py

```diff
@@ -286,7 +281,7 @@ def compute_permutation_importance(self, X, y, features: list, preprocess=True)
             x = [X.copy() for _ in range(compute_count)]  # TODO Make this much faster, only make this and concat it once. Then just update values and reset the values edited each iteration
```

Comment:
will this cause memory issue?

Reply:
On a large dataset, yes. I have a TODO mentioning that feature importance is not yet memory safe. I plan to add a memory check on X size to pick the proper value for compute_count

Reply:
Without compute_count (such as setting compute_count=1), the code is much slower